packages/data-service-generator/scripts/package-container.sh

```diff
@@ -21,5 +21,7 @@ if [[ $PRODUCTION_TAGS = "true" ]]; then
 
     npx nx internal:package:container data-service-generator --prod
 else
-    npx nx internal:package:container data-service-generator 
+    echo "Local deployment detected"
+    echo "Using Dockerfile-dev to include latest code-gen-types"
+    npx nx internal:package:container data-service-generator --dev
```

Comment:
`--dev` is an invalid option. Only production configuration has the alias `--prod`

```suggestion
    npx nx internal:package:container data-service-generator --configuration dev
```